src/mediator/auctionmediator.js

```diff
@@ -1028,5 +1028,16 @@ AuctionMediator.prototype.throwErrors = function(silent) {
   return this.throwErrors_;
 };
 
+/**
+ * Get the [AuctionMediator]{@link pubfood#mediator.AuctionMediator} event emitter instance.
+ * <br><br>  The singleton {@link PubfoodEvent} emitter reference is shared among
+ * {@link pubfood} objects.
+ * @returns {PubfoodEvent} the {@link pubfood} emitter singleton reference
+ * @private
+ */
+AuctionMediator.prototype.getEventEmitter = function() {
```

Comment:
While the `pubfood.removeAllListeners()` and `pubfood.removeListener()` functions do return the `PubfoodEvent` singleton; it is good measure here to encapsulate the intent for the event emitter to be private to the library.